src/NServiceBus.AwsLambda.SQS.AcceptanceTests/NativeIntegration/When_receiving_a_native_message_with_encoding.cs

```diff
@@ -109,14 +112,20 @@ public async Task Should_preserve_message_attributes_in_error_queue()
             var poisonMessages = await RetrieveMessagesInErrorQueue();
             var message = poisonMessages.Records[0];
 
-            Assert.That(poisonMessages.Records.Count, Is.EqualTo(1));
-            Assert.That(message, Is.Not.Null);
+            Assert.Multiple(() =>
+            {
+                Assert.That(poisonMessages.Records.Count, Is.EqualTo(1));
+                Assert.That(message, Is.Not.Null);
+            });
```

Comment:
For the future: message and poisonMessages should probably not be grouped

There are other fixes that have similar small issues